pkg/store/storepb/rpc.proto

```diff
@@ -42,9 +42,14 @@ enum StoreType {
 
 message InfoResponse {
   repeated Label labels = 1 [(gogoproto.nullable) = false];
```

Comment:
This one should be marked as deprecated no?

Reply:
something like `reserved 0;`

Reply:
:+1:

Reply:
Commented as deprecated.